src/ansys_sphinx_theme/__init__.py

```diff
@@ -372,6 +419,26 @@ def download_file(url: str, directory: pathlib.Path) -> None:
             file.write(response.content)
 
 
+def replace_html_tag(app, exception):
+    """Replace HTML tags in the generated HTML files.
+
+    Parameters
+    ----------
+    app : ~sphinx.application.Sphinx
+        Application instance for rendering the documentation.
+    exception : Exception
+        Exception that occurred during the build process.
+    """
+    if exception is None:
+        build_dir = pathlib.Path(app.builder.outdir).resolve()
+        file_names = [str(file) for file in build_dir.rglob("*.html")]
+        for file_name in file_names:
+            with open(build_dir / file_name, "r", encoding="utf-8") as f:
+                content = f.read()
+            with open(build_dir / file_name, "w", encoding="utf-8") as f:
+                f.write(content.replace("&lt;", "<").replace("&gt;", ">"))
```

Comment:
Some tips:

- Try to reduce indentation as much as possible. This can usually be achieved by checking the opposite condition.

  ```python
  if exception is not None:
      return
  ```

I would force this to run only in the HTML files located in an `api/` dir.

Reply:
> Some tips:
> 
> * Try to reduce indentation as much as possible. This can usually be achieved by checking the opposite condition.
>   ```python
>   if exception is not None:
>       return
>   ```
> 
> I would force this to run only in the HTML files located in an `api/` dir.

sure, thanks jorge, i will try to adapt this.